cadence/src/client.rs

```diff
@@ -77,17 +100,29 @@ impl ToTimerValue for Duration {
 ///
 /// Typical use of Cadence shouldn't require interacting with this trait.
 pub trait ToGaugeValue {
-    fn try_to_value(self) -> MetricResult<MetricValue>;
+    fn try_to_value(self) -> MetricResult<Vec<MetricValue>>;
 }
 
 impl ToGaugeValue for u64 {
-    fn try_to_value(self) -> MetricResult<MetricValue> {
-        Ok(MetricValue::Unsigned(self))
+    fn try_to_value(self) -> MetricResult<Vec<MetricValue>> {
+        Ok(vec![MetricValue::Unsigned(self)])
     }
 }
 impl ToGaugeValue for f64 {
-    fn try_to_value(self) -> MetricResult<MetricValue> {
-        Ok(MetricValue::Float(self))
+    fn try_to_value(self) -> MetricResult<Vec<MetricValue>> {
+        Ok(vec![MetricValue::Float(self)])
+    }
+}
+
+impl ToGaugeValue for Vec<u64> {
+    fn try_to_value(self) -> MetricResult<Vec<MetricValue>> {
+        Ok(self.into_iter().map(MetricValue::Unsigned).rev().collect())
```

Comment:
Why the call to `.rev()` here and elsewhere?

Reply:
whoops